tools/config.ts

```diff
@@ -59,7 +60,10 @@ export const NPM_DEPENDENCIES = [
 
 // Declare local files that needs to be injected
 export const APP_ASSETS = [
-  { src: `${ASSETS_SRC}/main.css`, inject: true, dest: CSS_DEST }
+  // inject minified css files from tmp
+  { src: `${TMP_ASSETS_DIR}/main.css`, inject: true, dest: CSS_DEST },
```

Comment:
The problem is that the CSS gets minified in the 'build.html_css.prod' task and ends up in the 'tmp' assets folder. The 'build.index' then copies the unminified CSS from 'ASSETS_SRC'.